Starting Debug for node-3
SIOCSARP: Invalid argument
arp: cannot set entry on line 3 of etherfile /home/vignesh/Desktop/emane-Timekeeper/conf/experiment/arp_table.txt !
Running Command /home/vignesh/Desktop/PLCs/awlsim-0.42/tests/run.sh --node 1 /home/vignesh/Desktop/PLCs/awlsim-0.42/tests/modbus/simple_client_server/sfb102_client.awl
48 (process ID) old priority 0, new priority 10
=== Running tests with 'python3'
Running test 'sfb102_client.awl' with 'python3' ... ['lxc.utsname ', ' lxc0-0']
['lxc.network.type ', ' veth']
['lxc.network.flags ', ' up']
['lxc.network.link ', ' br-0-0']
['lxc.network.ipv4 ', ' 10.10.0.9/8']
Dest IP :  10.10.0.9
Resolved hostname =  10.10.0.9
~~~~~~~~~~~~~~~~~~~~Inputs~~~~~~~~~~~~~~~~~~~
CONNECTION ID     =  2
CONN_TIME         =  5.0
RECV_TIME         =  5.0
DICONNECT         =  1
ENQ_ENR           =  1

~~~~~~~~~~~~~~~~~~~Outputs~~~~~~~~~~~~~~~~~~~
10.10.0.9 502
[Errno 101] Network is unreachable
DONE_NDR         =  True
LICENSED         =  True
CONN_ESTABLISHED =  False
BUSY             =  False
ERROR            =  True
STATUS_MODBUS    =  0
STATUS_CONN      =  a100
#############################################
[OK: 0:00.27]

All tests succeeded (full run)
